src/Stratis.Bitcoin.Features.RPC/RPCAuthorization.cs

```diff
@@ -42,6 +42,10 @@ public bool IsAuthorized(IPAddress ip)
 
             if (this.AllowIp.Count == 0)
                 return true;
+
+            if (this.AllowIp.Any(i => i.ToString() == "0.0.0.0"))
```

Comment:
@quantumagi lets extend this to also facilitate the usage of ranges.

Reply:
Done.